packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -86,3 +246,105 @@ export interface ListUnspentQueryOptions {
   minimumSumAmount?: number
   tokenId?: string
 }
+
+export interface CreateWalletOptions {
+  blank?: boolean
+  passphrase?: string
+  avoidReuse?: boolean
+}
+
+export interface SendToAddressOptions {
+  comment?: string
+  commentTo?: string
+  subtractFeeFromAmount?: boolean
+  replaceable?: boolean
+  confTarget?: number
+  estimateMode?: Mode
+  avoidReuse?: boolean
+}
+
+export interface IWallet {
```

Comment:
I think better to call it `CreateWalletResponse` or `WalletCreateResult` or `CreateWalletResult` or `...`